libraries/common/constants/Portals.js

```diff
@@ -66,9 +67,14 @@ export const PAGE_LOGIN_REGISTER_LINK_BEFORE = `${PAGE}.${LOGIN}.${REGISTER_LINK
 export const PAGE_LOGIN_REGISTER_LINK = `${PAGE}.${LOGIN}.${REGISTER_LINK}`;
 export const PAGE_LOGIN_REGISTER_LINK_AFTER = `${PAGE}.${LOGIN}.${REGISTER_LINK}.${AFTER}`;
 
+export const NAV_MENU_CONTENT = `${NAV_MENU}.${CONTENT}`;
```

Comment:
This isn't required anymore since we implement the portals inside of the quick links component